src/pages/product-details/detail/tests/detail.test.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import Adapter from 'enzyme-adapter-react-16';
+import Enzyme, { shallow } from 'enzyme';
+import Detail from '../detail';
+
+jest.mock('../detail.styles', () => ({
+  useStyles: () => ({})
+}));
+
+jest.mock('react-redux');
+Enzyme.configure({ adapter: new Adapter() });
```

Comment:
Try to remove "adapter" and run test
If everything will be ok, remove it from all your tests